src/components/dashboard/ReceiveConfirmation.js

```diff
@@ -46,7 +47,7 @@ const ReceiveConfirmation = ({ screenProps, styles }: ReceiveProps) => {
     return generateShareLink('receive', code)
   }, [code, generateShareLink])
 
-  const sharePressHandler = useCallback(() => {
+  const sharePressHandler = useOnPress(() => {
```

Comment:
if setString comes from the other hook (i suppose useClipboard ?) - add it to the deps too